client/enduser/src/main/java/org/apache/syncope/client/enduser/resources/UserSelfUpdateResource.java

```diff
@@ -57,8 +63,8 @@ protected ResourceResponse newResourceResponse(final IResource.Attributes attrib
                 throw new IllegalArgumentException("Entered captcha is not matching");
             }
 
-            UserTO userTO = MAPPER.readValue(request.getReader().readLine(), UserTO.class);
-            LOG.debug("User {} id self-updating", userTO.getUsername());
+            String json = request.getReader().readLine();
```

Comment:
Don't see any reason for this change - and debug statement also went away: please revert this change.